stac_fastapi/extensions/stac_fastapi/extensions/core/collection_search/collection_search.py

```diff
@@ -46,7 +52,7 @@ class CollectionSearchExtension(ApiExtension):
             the extension
     """
 
-    GET = CollectionSearchExtensionGetRequest
+    GET: BaseCollectionSearchGetRequest = attr.ib(default=BaseCollectionSearchGetRequest)
```

Comment:
⚠️ this makes the `GET` attribute *editable* on user input. 

This attribute isn't really used externally or internally because the model is passed to the application using `collections_get_request_model` option, but it make sense